src/planscape/forsys/forsys_request_params.py

```diff
@@ -157,6 +157,33 @@ def __init__(self, request: HttpRequest):
             DbRequestParams._URL_WRITE_TO_DB, True)
         self.scenario = get_scenario_by_id(
             self.user, self._URL_SCENARIO_ID, params)
+        
+
+# Parameters for deciding whether a stand is eligible for treatment.
+# TODO: make it possible to parse these parameters from URL and/or read them 
+# from the DB.
+class StandEligibilityParams:
+    BUILDINGS_KEY = "buildings"
+    ROAD_PROXIMITY_KEY = "road_proximity"
+    SLOPE_KEY = "slope"
+
+    filter_by_buildings: bool
+
+    filter_by_slope: bool
+    max_slope_in_percent_rise: float
+
+    filter_by_road_proximity: bool 
+    max_distance_from_road_in_meters: float
+
+    def __init__(self):
+        self.filter_by_buildings = False
+        self.filter_by_slope = False 
+        # This corresponds with 35 degrees.
+        self.max_slope_in_percent_rise = 70.0
+        self.filter_by_road_proximity = False 
+        # Advised to set this to ~60 meters, but that's not testable for 300m 
```

Comment:
add a TODO to change this after 30m data is available?

Reply:
Done.